src/Raven.Server/Documents/Handlers/StatsHandler.cs

```diff
@@ -41,6 +41,14 @@ public Task Stats()
                 var attachments = Database.DocumentsStorage.AttachmentsStorage.GetNumberOfAttachments(context);
                 stats.CountOfAttachments = attachments.AttachmentCount;
                 stats.CountOfUniqueAttachments = attachments.StreamsCount;
+
+                using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext serverContext))
+                using (serverContext.OpenReadTransaction())
+                {
+                    stats.CountOfIdentities = ServerStore.Cluster.GetNumberOfIdentities(serverContext, Database.Name);
```

Comment:
@ayende I'm wondering if we should not move it from DatabaseStatistics to:
a) separate endpoint per each of those
b) to endpoint like /databases/stats/detailed

Reply:
I think `/detailed` is better yes.
In 3.5 we had a stats endpoint just for the studio to hit, because it `/stats` was getting expensive.